openpower/platform/Config.in

```diff
@@ -2,3 +2,11 @@ config BR2_OPENPOWER_PLATFORM
     bool "Enable the OpenPower Platform"
     default n
     depends on BR2_powerpc64 || BR2_powerpc64le
+
+config BR2_OPENPOWER_POWER8
```

Comment:
hm, the best way I can think of doing this in a one liner is 


if ((BR2_OPENPOWER_POWER8 || BR2_OPENPOWER_POWER9) && !(BR2_OPENPOWER_POWER8 && BR2_OPENPOWER_POWER9))

which will get messy when added to 9 files and then need updating for future chips.
Thoughts on adding the BR2_OPENPOWER_PLATFORM config back and then having it depend on the above equation. And then all the checks will just be for BR2_OPENPOWER_PLATFORM